services/skus/datastore_test.go

```diff
@@ -151,6 +151,24 @@ func (suite *PostgresTestSuite) TestGetOrderByExternalID() {
 	}
 }
 
+func (suite *PostgresTestSuite) TestCountActiveOrderCreds_Success() {
```

Comment:
suggestions(non-blocking): this test doesn't follow the new pattern and we are also missing several conditions such as expired and tokens.

Add the extra conditions and refactor to the new pattern